content/departments/product-engineering/engineering/enablement/dev-experience/processes.md

```diff
@@ -0,0 +1,159 @@
+
+We inherit Sourcegraph's [meeting principles](../../../../../company-info-and-process/communication/index.md#internal-meetings) and [asynchronous communication guidelines](../../../../../company-info-and-process/communication/asynchronous-communication.md#how-to-choose-sync-vs-async) with some modifications that help us run effective meetings:
+
+- Meeting leaders rotate weekly, and the leader should prepare the agenda at least 24 hours ahead of time.
+- All teammates should add items to the agenda before the meeting begins.
+- Any action items that come out of meetings should be considered a high priority.
+
+### Team Sync
+
+We connect weekly to discuss any topics that are best addressed in person. Our notes are available [here](https://docs.google.com/document/d/1Lm6GT-F4v9OTa5wxa1-AKLtNwlDkORbbeGjqVd9kWPg/edit).
+
+### Team Retro
+
+We connect bi-weekly to celebrate wins from the past two weeks, give cheers to our teammates, and reflect on what we’ve liked, not liked, and learned for going forward. The focus of the [agenda](https://docs.google.com/document/d/1QR1It6KGccwWRpASH16J64QNkpHtngI4o2ttrGpVCwU/edit#) for this meeting is team health, as opposed to day-to-day work.
+
+### Coffee
+
+Every week we hold an informal, totally optional coffee break to connect socially and hang out together as humans.
+
+### `sg` hack hour
+
+There is a weekly `sg` hack hour that Thorsten Ball and the DevX team co-host weekly Fridays from 16:00 UTC to 17:00 UTC for anyone interested in making contributions to [the Sourcegraph developer tool](https://docs.sourcegraph.com/dev/background-information/sg).
+
+When the hack hour starts, a meeting link will be posted in #dev-experience.
+
+To learn more about contributing to `sg`, check out the [contribution guide](https://docs.sourcegraph.com/dev/background-information/sg#contributing-to-sg)!
+
+### Slack
+
+Teammates communicate in #dev-experience, #dev-experience-internal, and #dev-experience-private, and post daily updates [Geekbot](https://app.geekbot.com/dashboard/standup/90468/view/insights) in #dev-experience-updates.
+
+## Housekeeping
+
+### Weekly reminders
+
+At the start of each week, the PM posts a reminder in `#dev-experience-internal` indicating who is on support and who is leading meetings.
+
+### Updating the Dev Experience project board
+
+All engineers update the board in advance of the weekly sync.
+
+### Updating the Product and Engineering roadmap
+
+The PM and work leads own the roadmap issues. They solicit input from the team to make sure the problem space is thorough, measures of success are ambitious-but-reasonable, and technical details are correct.
+
+### Updating our Strategy page
+
+The PM updates this page at the start of each month.
+
+### Newsletter
+
+The DevX team is responsible for a **monthly** (ish) newsletter to highlight developer experience updates (not just those lead by the team). Learn more about it and see previous issues [in the newsletter archive](newsletter.md).
```

Comment:
oops hehe definitely hasn't been monthly 😬 would love to bring this back soon!

Reply:
We say "monthly (ish)" 😂 !